### PR TITLE
Adding robot_process to indigo

### PR DESCRIPTION
**Pull request by asiron**

### CODE REVIEW

**Comment by mikaelarguedas** [comment]
Cool stuff !
This seems similar to the managed nodes introduced in ROS 2. Maybe we could converge in the future on the state machine and ROS interfaces, some references if you're interested: [design doc](http://design.ros2.org/articles/node_lifecycle.html), [ROS interfaces](https://github.com/ros2/rcl_interfaces/tree/master/lifecycle_msgs)

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -11242,6 +11242,17 @@ repositories:
       url: https://github.com/GT-RAIL/robot_pose_publisher.git
       version: develop
     status: maintained
+  robot_process:
+    doc:
+      type: git
+      url: https://github.com/snt-robotics/robot_process.git
+      version: master
+    source:
+      test_pull_requests: true
```
Comment (mikaelarguedas) [inline indigo/distribution.yaml]: Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, please either remove this line or provide the necessary access as described in the wiki page.

Comment (asiron) [reply indigo/distribution.yaml]: Sorry, I forgot about this step. I generated a webhook as I can't give admin access to the org

Comment (mikaelarguedas) [reply indigo/distribution.yaml]: :+1:

**Review by mikaelarguedas — APPROVED** [review]
lgtm

**Pull request merged**